packages/key-management/src/types.ts

```diff
@@ -175,9 +177,11 @@ export interface KeyAgent {
 
 export type AsyncKeyAgent = Pick<KeyAgent, 'deriveAddress' | 'derivePublicKey' | 'signBlob' | 'signTransaction'> & {
   knownAddresses$: Observable<GroupedAddress[]>;
+  getKeyAgent(): Promise<KeyAgent>;
   getChainId(): Promise<Cardano.ChainId>;
   getBip32Ed25519(): Promise<Crypto.Bip32Ed25519>;
   getExtendedAccountPublicKey(): Promise<Crypto.Bip32PublicKeyHex>;
+  setKnownAddresses(addresses: GroupedAddress[]): void;
```

Comment:
All methods in **Async**KeyAgent must be async
```suggestion
  setKnownAddresses(addresses: GroupedAddress[]): Promise<void>;
```